Traceback (most recent call last):
  File "C:/Users/Sam Mallet/Desktop/Debugging/Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
Traceback (most recent call last):
  File "C:/Users/Sam Mallet/Desktop/Debugging/Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
Traceback (most recent call last):
  File "C:\Users\Sam Mallet\Desktop\Debugging\Raise&Assert.py", line 34, in <module>
    raise Exception('This is the error message.') # makes generic error message
Exception: This is the error message.
